Project/Source/Resources/ResourceTexture.cpp

```diff
@@ -39,19 +39,12 @@ void ResourceTexture::Load() {
 
 	// Load image
 	ilBindImage(image);
-	bool imageLoaded = ilLoad(IL_DDS, filePath.c_str());
+	bool imageLoaded = ilLoad(IL_RAW, filePath.c_str());
 	if (!imageLoaded) {
 		LOG("Failed to load image.");
 		return;
 	}
 
-	// Flip image if neccessary
-	ILinfo info;
-	iluGetImageInfo(&info);
-	if (info.Origin == IL_ORIGIN_UPPER_LEFT) {
-		iluFlipImage();
-	}
```

Comment:
Why is this removed¿

Reply:
Because RAW images don't save orientation information, so I had to change the place where it gets flipped.